Makefile

```diff
@@ -23,7 +23,7 @@ clean:
 	rm -rf vendor
 
 test: deps
-	@go test $(BUILDFLAGS) -cover $(PACKAGES)
+	@go test $(BUILDFLAGS) -cover $(PACKAGES) ./...
```

Comment:
I copied over this repository so that I could test directly with Skopeo.  I agree it is a weird case, and not something you probably want to support, so feel free to close this PR